### DISCUSSION TITLE
How do I see what is using all my shared storage?

### DISCUSSION THREAD

**Discussion by ali1234** [community/community#62940] [Actions] [answered]
### Select Topic Area

Question

### Body

I don't use Packages and I don't even know what they are.

All my workflows are set to expire artefacts after 7 days.

How do I find out what is using the space?

![image](https://github.com/community/community/assets/651970/164f587c-1a70-480c-a2e4-ba66c700fd44)

**Comment by mehulrajdev** [comment]
In the upper-right corner of any page, click your profile photo, then click Settings.

Screenshot of GitHub's account menu showing options for users to view and edit their profile, content, and settings. The menu item "Settings" is outlined in dark orange.

In the "Access" section of the sidebar, click  Billing and Plans, then click Plans and Usage.

Under "GitHub Packages", view details of your usage for data transfer.

Under "Storage for Actions and Packages", view details of your storage usage for GitHub Actions and GitHub Packages combined. Storage usage data synchronizes every hour.

Optionally, next to "Usage this month", click Get usage report to email a CSV report of storage used for GitHub Actions and GitHub Packages to the account's primary email address.

**Comment by ali1234** [comment]
![image](https://github.com/community/community/assets/651970/f4a6c96d-cdf0-48c9-b4fc-d7d1b1cdccfb)

Usage and Billing doesn't say what is using the space.

Neither does the detailed usage report:

```
2023-07-30,Shared Storage,Shared Storage,0.0774,gb-day,0.008,1.0,ali1234,blockgame,,,
2023-07-31,Shared Storage,Shared Storage,0.8053,gb-day,0.008,1.0,ali1234,blockgame,,,
```

The usage suddenly increased by 700MB for no reason. And this is still less than the supposed 1800MB that I am actually using.

**Comment by ali1234** [comment, marked as answer]
Github Support told me this is a problem on their side.
